src/components/Restoration/RestorationType.vue

```diff
@@ -65,71 +79,58 @@ import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restora
   }
 })
 export default class RestorationType extends Vue {
-  @Getter getRestoration!: RestorationStateIF
   @Getter getCurrentDate!: string
+  @Getter getRestoration!: RestorationStateIF
   @Getter getRestorationTypeValid!: boolean
+  @Getter getShowErrors!: boolean
 
-  @Action setRestorationType!: ActionBindingIF
   @Action setRestorationExpiry!: ActionBindingIF
   @Action setRestorationRelationships!: ActionBindingIF
+  @Action setRestorationType!: ActionBindingIF
   @Action setRestorationTypeValid!: ActionBindingIF
 
   // Local properties
-  protected selectRestorationType:RestorationTypes = null
+  protected selectRestorationType: RestorationTypes = null
 
+  // Enum for template
   readonly RestorationTypes = RestorationTypes
 
-  /**
-   * Called when component is mounted.
-   * Automatically check limited or full restoration when user continues a draft as per last selection.
-   */
-  mounted (): void {
-    if (this.getRestoration.type === RestorationTypes.LIMITED) {
-      this.selectRestorationType = RestorationTypes.LIMITED
-    } else {
-      this.selectRestorationType = RestorationTypes.FULL
-    }
-  }
-
-  // Computed value getter that returns true if current restoration is full
+  /** Is true if current restoration is full. */
   get isFull (): boolean {
     return (this.getRestoration.type === RestorationTypes.FULL)
   }
 
-  // Computed value getter that returns true if current restoration is limited
+  /** Is true if current restoration is limited. */
   get isLimited (): boolean {
     return (this.getRestoration.type === RestorationTypes.LIMITED)
   }
 
+  /** This section's validity state (when prompted by app). */
+  get invalidSection (): boolean {
+    return (this.getShowErrors && !this.getRestorationTypeValid)
+  }
+
   /**
-   * Set the selected Limited Restoration choice
-   * @param val The value of the selected radio button
+   * Sets the selected Limited Restoration choice.
+   * @param val the value of the selected radio button
    */
-  @Watch('selectRestorationType')
-  private setRestorationChoice (val) {
-    this.setRestorationType(val)
+  protected changeRestorationType (): void {
+    this.setRestorationType(this.selectRestorationType)
     if (this.isLimited) {
       this.setRestorationRelationships([])
-      this.setRestorationTypeValid(true)
     } else {
       this.setRestorationExpiry(null)
     }
   }
 
-  @Watch('$route')
-  private async scrollToInvalidComponent (): Promise<void> {
-    if (this.$route.name === RouteNames.RESTORATION_BUSINESS_NAME) {
```

Comment:
This is done in the parent, consistent with all other step components (@JazzarKarim ).